package.json

```
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@nuxt/kit": "^3.7.3",
-    "primevue": "~3.49.1"
+    "primevue": "~3.51.0"
```

Current approach of allowing patch-only updates might be too restrictive and relies on maintainers to act, potentially locking end users from getting the latest updates as soon as they get released.
```suggestion
    "primevue": "^3.51.0"
```

---

I agree this would be the better approach but wasn't sure if there is a specific reasoning behind version pinning so I kept that as is, I'll leave it up to the maintainers whether this should be more permissive. 

/cc @mertsincan